routes/index.js

```diff
@@ -1,15 +1,17 @@
 const userRoutes = require("./api/user.routes.js");
 const componentRoutes = require("./component.routes.js");
 const appRoutes = require("./app.routes.js");
+const messageRoutes = require("./api/conversations.routes");
 
 const constructorMethod = (app) => {
   app.use("/api/user", userRoutes);
   app.use("/load", componentRoutes);
   app.use("/", appRoutes);
+  app.use("/sync", messageRoutes);
```

Comment:
use /api/ROUTE_NAME instead here

Reply:
@jvall0228 Committed and pushed recommended changes.